Configs/.config/hypr/scripts/cpuinfo.sh

```diff
@@ -1,35 +1,80 @@
-#!/usr/bin/env sh
+#!/bin/bash
 
-# CPU model
-model=$(lscpu | awk -F ':' '/Model name/ {sub(/^ *| *$/,"",$2); print $2}' | awk '{NF-=3}1')
 
-# CPU utilization
-utilization=$(top -bn1 | awk '/^%Cpu/ {print 100 - $8}')
+# Thee shalt find the greatest one,
+# He who not more than the chosen one
+map_floor() {
 
-# Clock speed
-freqlist=$(cat /proc/cpuinfo | grep "cpu MHz" | awk '{ print $4 }')
-maxfreq=$(lscpu | grep "CPU max MHz" | awk -F: '{ print $2}' | sed -e 's/ //g' -e 's/\.[0-9]*//g')
-frequency=$(echo $freqlist | tr ' ' '\n' | awk "{ sum+=\$1 } END {printf \"%.0f/$maxfreq MHz\", sum/NR}")
+    # From the depths of the string, words arise,
+    # Keys in pairs, a treasure in disguise.
+    IFS=', ' read -r -a pairs <<< "$1"
 
-# CPU temp
-temp=$(sensors | awk '/Package id 0/ {print $4}' | awk -F '[+.]' '{print $2}')
-if [ -z "$temp" ]; then
-    temp=$(sensors | awk '/Tctl/ {print $2}' | tr -d '+°C')
-fi
-if [ -z "$temp" ]; then
-    temp="N/A"
-fi
+    # If the final token stands alone and bold,
+    # Declare it the default, its worth untold.
+    if [[ ${pairs[-1]} != *":"* ]]; then
+        def_val="${pairs[-1]}"
+        unset 'pairs[${#pairs[@]}-1]'
+    fi
+
+    # Scans the map, a peak it seeks,
+    # The highest passed, the value speaks.
+    for pair in "${pairs[@]}"; do
+        IFS=':' read -r key value <<< "$pair"
+        
+        # Behold! Thou holds the secrets they seek,
+        # Declare it and silence the whispers unique.
+        if [ ${2%%.*} -gt $key ]; then
+            echo "$value"
+            return
+        fi
+    done
 
-# map icons
-set_ico="{\"thermo\":{\"0\":\"\",\"45\":\"\",\"65\":\"\",\"85\":\"\"},\"emoji\":{\"0\":\"❄\",\"45\":\"☁\",\"65\":\"\",\"85\":\"\"},\"util\":{\"0\":\"󰾆\",\"30\":\"󰾅\",\"60\":\"󰓅\",\"90\":\"\"}}"
-eval_ico() {
-    map_ico=$(echo "${set_ico}" | jq -r --arg aky "$1" --argjson avl "$2" '.[$aky] | keys_unsorted | map(tonumber) | map(select(. <= $avl)) | max')
-    echo "${set_ico}" | jq -r --arg aky "$1" --arg avl "$map_ico" '.[$aky] | .[$avl]'
+    # On this lonely shore, where silence dwells
+    # Even the waves, echoes words unheard
+    [ -n "$def_val" ] && echo $def_val || echo " "
 }
 
-thermo=$(eval_ico thermo $temp)
-emoji=$(eval_ico emoji $temp)
-speedo=$(eval_ico util $utilization)
+# Define glyphs
+if [[ $NO_EMOJI -eq 1 ]]; then
+    temp_lv="85:, 65:, 45:☁, ❄"
+else
+    temp_lv="85:🌋, 65:🔥, 45:☁️, ❄️"
+fi
+util_lv="90:, 60:󰓅, 30:󰾅, 󰾆" 
+
+# Get static CPU information
+model=$(lscpu | awk -F': ' '/Model name/ {gsub(/^ *| *$| CPU.*/,"",$2); print $2}')
```

Comment:
`lscpu` respects `LANG` and thus requires `en_us` for this to work.

Reply:
Great, good find. I'll add this on the next commit.